samples/distro-examples/tests/output/all.out

```diff
@@ -103,7 +103,7 @@ ATANH:nan
 ATN:1.489673934694
 BCS:catsanddogs
 BGETC:
-BIN:00000000000000000000000000001100
+BIN:1100
```

Comment:
It would be good to add some more testing for BIN. Something like this could be added to strings.bas

```
bin_test=[[,], [0, "0"], [7, "111"], [14, "1110"], [15, "1111"]]
for t in bin_test
  if (t[1] != bin(t[0])) then throw "err:" + str(t) + " <> " + bin(t[0])
next  
```